src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+       action="https://mate-academy-form-lesson.herokuapp.com/ create-application"    
+       method="post">
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/ create-application"    
      method="post"
    >
```
Closing tag should be under the opening. Also, two spaces indentation is enough(not 3). Please fix all cases